src/main/java/step3/service/RacingGame.java

```diff
@@ -0,0 +1,38 @@
+package step3.service;
+
+import step3.domain.Car;
+import step3.domain.Cars;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+
+    private final int numberOfCars;
+    private final int attemps;
+
+    public RacingGame(int numberOfCars, int attemps) {
+        this.numberOfCars = numberOfCars;
+        this.attemps = attemps;
+    }
+
+    public List<Car> setCars() {
+        List<Car> carList = new ArrayList<>();
+        for (int i = 0; i < numberOfCars; i++) {
+            carList.add(new Car());
+        }
+        return carList;
+    }
```

Comment:
```suggestion
    public List<Car> createCars() {
        List<Car> carList = new ArrayList<>();
        for (int i = 0; i < numberOfCars; i++) {
            carList.add(new Car());
        }
        return carList;
    }
```

매우 띠용했던 메서드 명입니다. 🙄 
일반적으로 set 메서드는 리턴이 void 이고, 파라미터를 받아서 객체 내부의 멤버를 바꿔주는 역할을 수행합니다.
즉 메서드명만 보면 `void setCars(List<Car> cars)` 같은 느낌을 줍니다.
하지만 실제로는 carList 를 생성해서 반환하는 역할을 하고 있군요.
이런 메서드를 **팩토리 매서드** 라고 합니다.
한번 Factory 디자인 패턴에 대해서 공부해보시면 좋을 것 같습니다.

리턴타입도 `List<Car>` 가 아니라, 이미 정의해놓은 일급 컬렉션인 `Cars` 이면 좋겠네요.

Reply:
앗 메서드명에서 그렇게까지 생각하진 못했네요..!
수정하겠습니다 반환타입도 일급 컬렉션을 만들어두고 사용하질 않고있었네요.